platform/mock-local-home-platform.ts

```diff
@@ -97,27 +45,26 @@ export class MockLocalHomePlatform implements MockUDPListener {
     return MockLocalHomePlatform.instance;
   }
 
-  public getDeviceManager(): smarthome.DeviceManager {
-    return this.deviceManager;
-  }
-
-  public getLocalDeviceIdMap(): Map<string, string> {
-    return this.localDeviceIds;
+  private onNewDeviceIdRegistered(localDeviceId: string) {
+    this.newDeviceRegisteredActions.forEach((newDeviceRegisteredAction) => {
+      newDeviceRegisteredAction(localDeviceId);
+    });
   }
 
-  private setupUDP() {
-    this.udpScanConfigs.forEach((udpScanConfig) => {
-      this.mockNetwork.registerUDPListener(
-        this,
-        udpScanConfig.listenPort,
-        udpScanConfig.broadcastAddress
-      );
+  public async getNextDeviceIdRegistered(): Promise<string> {
+    return new Promise((resolve) => {
+      this.newDeviceRegisteredActions.push((localDeviceId) => {
+        resolve(localDeviceId);
+      });
     });
   }
 
-  // Establish fulfillment path using app code
-  async onUDPMessage(msg: Buffer, rinfo: RemoteAddressInfo): Promise<void> {
-    console.log('received discovery payload:', msg, 'from:', rinfo);
+  /**
+   * Takes a `discoveryBuffer` and passes it to the fulfillment app in an `IdentifyRequest`
+   * @param discoveryBuffer
```

Comment:
Typically a JSDoc parameter includes a short description of the parameter.

Reply:
Done.